scripts/leapfrog/ubuntu14-leapfrog.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+# (c) 2017, Jean-Philippe Evrard <jean-philippe.evrard@rackspace.co.uk>
+
+## Shell Opts ----------------------------------------------------------------
+set -e -u -x
+set -o pipefail
+
+## Standard Vars --------------------------------------------------------------
+export OA_OVERRIDES='/etc/openstack_deploy/user_osa_variables_overrides.yml'
+export OA_DEFAULTS='/etc/openstack_deploy/user_osa_variables_defaults.yml'
+export RPCD_OVERRIDES='/etc/openstack_deploy/user_rpco_variables_overrides.yml'
+export RPCD_DEFAULTS='/etc/openstack_deploy/user_rpco_variables_defaults.yml'
+export RPCD_SECRETS='/etc/openstack_deploy/user_rpco_secrets.yml'
+export RPCO_DEFAULT_FOLDER="/opt/rpc-openstack"
```

Comment:
Is there a reason that you aren't using the functions export that all the other scripts use to set these variables?